google-cloud-firestore/test/helper.rb

```diff
@@ -115,10 +143,13 @@ class MockFirestore < Minitest::Spec
   let(:firestore_mock) { Minitest::Mock.new }
 
   before do
+    # puts "#{self.class.name}"
```

Comment:
guess these lines(time elapsed puts) can be removed ?

Reply:
Updated.